requirements.txt

```diff
@@ -1,26 +1,27 @@
--i https://pypi.org/simple/
+-i https://pypi.org/simple
 alabaster==0.7.12
 babel==2.7.0
 certifi==2019.9.11
 chardet==3.0.4
-docutils==0.15.2
-idna==2.8
-imagesize==1.1.0
-jinja2==2.10.3
-markupsafe==1.1.1
-packaging==19.2
-pygments==2.4.2
-pyparsing==2.4.2
+docutils==0.15.2; python_version != '3.1.*'
```

Comment:
I don't know what is this, but it doesn't matter, because I want to migrate this to [RTD v2](https://docs.readthedocs.io/en/latest/config-file/v2.html) as a next step and the lockfile will thus soon change to something different anyway.